page/sass/common.scss

```diff
@@ -55,11 +55,12 @@ body{
         padding: 0 0 $space-bottom-between-elements $space-bottom-between-elements;
         li {
             color: black;
-            font-weight: $font-weight-bold;
+            font-weight: $font-weight-bold;            
```

Comment:
what kind of editor you use? you're leaving a lot of trash like this in source code